genie-web/src/test/groovy/com/netflix/genie/web/agent/resources/AgentFileProtocolResolverSpec.groovy

```diff
@@ -46,7 +47,7 @@ class AgentFileProtocolResolverSpec extends Specification {
         setup:
         String jobId = UUID.randomUUID().toString()
         Path relativePath = Paths.get("foo/bar.txt")
-        URI uri = new URI(AgentFileProtocolResolver.URI_SCHEME, jobId, "/" + relativePath, null)
+        URI uri = AgentFileProtocolResolver.createUri(jobId, "/" + relativePath)
```

Comment:
Are you always expecting the `path` to have the `/` as a start? I didn't see in the code if you're checking for it. What happens if it's not there?

Reply:
No, works in both cases. I'll add a test to document this behavior.

Reply:
Added